### DISCUSSION TITLE
.py to .qasam?

### DISCUSSION THREAD

**Discussion by InfamousPlatypus** [Spooky-Manufacturing/8Q#30] [Q&A]
In #29 I started working with some quantum and classical code. Um, I'm not sure how we were planning on doing that. I assume we were going to use Qasam but I'm not sure how we would be doing that. Could I have some guidance? 
Thanks!

**Comment by NoahGWood** [comment]
I've been using the OpenQASM language since that seems to be becoming industry standard, I'm open to suggestion though!

**Comment by InfamousPlatypus** [comment]
I guess I'm mostly asking if there was a way to call an OpenQASM function
(I guess those are called a custom gate?) from python. It's been a hot
second since I looked, so it may have changed with the newest spec, but the
only way I saw was qiskit.

Does this computer only have gate level controls or are pulse level (or
equivalent) controls possible?

Sorry if these are answered somewhere, I'm sort of limited to my phone and
it doesn't play well with github.

Reminder for me: Check new qasm specs, also can we create a backend for
qiskit?
